zombienets/xcmp.toml

```diff
@@ -0,0 +1,45 @@
+[settings]
+provider = "native"
+timeout = 1000
+
+[relaychain]
+default_command = "../polkadot/target/release/polkadot"
+chain = "rococo-local"
+
+  [[relaychain.nodes]]
+  name = "alice"
+
+  [[relaychain.nodes]]
+  name = "bob"
+
+[[parachains]]
+id = 2114
+cumulus_based = true
+chain = "turing-dev"
+
+  [parachains.collator]
+  name = "collator2114"
+  command = "./target/release/oak-collator"
+  ws_port = 9946
+
+[[parachains]]
+id = 1999
+cumulus_based = true
+chain = "local"
+
+  [parachains.collator]
+  name = "collator1999"
+  command = "../substrate-parachain-template/target/release/parachain-collator"
```

Comment:
they updated the node name to `parachain-template-node` in 0.9.25.
https://github.com/substrate-developer-hub/substrate-parachain-template/pull/121#discussion_r923463043